app-laptop/thinkfan/thinkfan-0.9.3.ebuild

```diff
@@ -18,17 +18,21 @@ IUSE="atasmart"
 DEPEND="atasmart? ( dev-libs/libatasmart )"
 RDEPEND="${DEPEND}"
 
-S="${WORKDIR}"
-
 src_prepare() {
+	default
+
+	sed -e "s:#!/sbin/runscript:#!/sbin/openrc-run:" \
+		-i rcscripts/thinkfan.gentoo
+
 	sed -e "s:share/doc/${PN}:share/doc/${P}:" \
```

Comment:
documentation usually should be installed in `share/doc/${PF}` (which is the same in this case)
On a slightly different note: could you try to get upstream to use the `GNUInstallDirs` module, which makes changing paths _much_ easier? See for instance how I converted jasper https://github.com/mdadams/jasper/pull/101/commits/10fb52dbcbacc70fdd014c5ee815d33e852d28b8 .

We can then specify `-DCMAKE_INSTALL_DOCDIR="${EPREFIX}"/usr/share/doc/${PF}` instead of constantly sed'ing the cmake files.